Gemfile

```diff
@@ -1,4 +1,7 @@
 source 'https://rubygems.org'
 
+gem 'activesupport'
```

Comment:
This is to allow me to use `#deep_symbolize_keys` and `#present?`

Reply:
could these go in the gemspec file instead?